encodeAndDecodeStrings.py

```diff
@@ -0,0 +1,45 @@
+class Codec:
+
+    def encode(self, strs):
+        """Encodes a list of strings to a single string.
+        
+        :type strs: List[str]
+        :rtype: str
+        """
+        string = ''
+        for str in strs:
+            for char in str:
+                if char == ":":
+                    string += "::"
+                elif char == ";":
+                    string += ";;"
+                else:
+                    string += char
+            string += ":;"
```

Comment:
Did not look at the question, but are we not allowed to use any other chars?

Reply:
I think so, if not, how can we decode the encoded string?